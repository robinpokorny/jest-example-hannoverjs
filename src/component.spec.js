import React from 'react'
import renderer from 'react-test-renderer'

import Component from './component'

test('renders without props', () => {
  const tree = renderer.create(
    <Component />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
