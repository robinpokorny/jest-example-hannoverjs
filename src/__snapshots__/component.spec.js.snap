// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders without props 1`] = `
<div>
  <a
    href="/item/0"
    onClick={[Function]}
  >
    Item No. 
    0
  </a>
  <a
    href="/item/1"
    onClick={[Function]}
  >
    Item No. 
    1
  </a>
  <a
    href="/item/2"
    onClick={[Function]}
  >
    Item No. 
    2
  </a>
  <a
    href="/item/3"
    onClick={[Function]}
  >
    Item No. 
    3
  </a>
  <a
    href="/item/4"
    onClick={[Function]}
  >
    Item No. 
    4
  </a>
  <a
    href="/item/5"
    onClick={[Function]}
  >
    Item No. 
    5
  </a>
  <a
    href="/item/6"
    onClick={[Function]}
  >
    Item No. 
    6
  </a>
  <a
    href="/item/7"
    onClick={[Function]}
  >
    Item No. 
    7
  </a>
  <a
    href="/item/8"
    onClick={[Function]}
  >
    Item No. 
    8
  </a>
  <a
    href="/item/9"
    onClick={[Function]}
  >
    Item No. 
    9
  </a>
</div>
`;
